src/Bindings/DeprecatedBindings.cpp

```diff
@@ -298,15 +298,15 @@ static int tolua_cWorld_SetSignLines(lua_State * tolua_S)
 	else
 	#endif
 	{
-		cWorld * self       = (cWorld *) tolua_tousertype (LuaState, 1, nullptr);
-		int BlockX          = (int)      tolua_tonumber   (LuaState, 2, 0);
-		int BlockY          = (int)      tolua_tonumber   (LuaState, 3, 0);
-		int BlockZ          = (int)      tolua_tonumber   (LuaState, 4, 0);
-		const AString Line1 =            tolua_tocppstring(LuaState, 5, 0);
-		const AString Line2 =            tolua_tocppstring(LuaState, 6, 0);
-		const AString Line3 =            tolua_tocppstring(LuaState, 7, 0);
-		const AString Line4 =            tolua_tocppstring(LuaState, 8, 0);
-		cPlayer * Player    = (cPlayer *)tolua_tousertype (LuaState, 9, nullptr);
+		cWorld * self       = reinterpret_cast<cWorld *>(tolua_tousertype (LuaState, 1, nullptr));
+		cPlayer * Player    = reinterpret_cast<cPlayer*>(tolua_tousertype (LuaState, 9, nullptr));
+		int BlockX          = static_cast<int>(tolua_tonumber(LuaState, 2, 0));
+		int BlockY          = static_cast<int>(tolua_tonumber(LuaState, 3, 0));
+		int BlockZ          = static_cast<int>(tolua_tonumber(LuaState, 4, 0));
+		const AString Line1 = tolua_tocppstring(LuaState, 5, 0);
+		const AString Line2 = tolua_tocppstring(LuaState, 6, 0);
+		const AString Line3 = tolua_tocppstring(LuaState, 7, 0);
+		const AString Line4 = tolua_tocppstring(LuaState, 8, 0);
```

Comment:
You are right about that, @madmaxoft . I was just getting so frustrated with the warnings that I just wanted them gone instead of looking at the real reason they were there. My future commits like this will be more focused on fixing rather than silencing.